README.md

```diff
@@ -25,6 +25,51 @@ public void ConfigureServices(IServiceCollection services)
     services.AddMvc();
 }
 ```
+For .NET 9 or later targets, the `SSMXmlRepository` that this package uses, has been updated to use `IDeletableXmlRepository` (an extension of `IXmlRepository`) from `Microsoft.AspNetCore.DataProtection.Repositories` namespace. `IDeletableXmlRepository` supports deletion of elements. While it is recommended not deleting data protection keys, in exceptional cases, such as extremely long-running services, applications might need to delete keys that are no longer in use and accepts the risk of data loss in exchange for storage savings. Below example demonstrates possible approach on how to delete keys that have expired within certain time frame.
+```csharp
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+using Microsoft.AspNetCore.DataProtection.KeyManagement.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddControllersWithViews();
+
+builder.Services.AddDataProtection()
+    .PersistKeysToAWSSystemsManager("/MyApplication/DataProtection");
+
+var app = builder.Build();
+
+var keyManager = app.Services.GetService<IKeyManager>();
+
+if (keyManager is XmlKeyManager xmlKeyManager)
```

Comment:
nit: Instead of casting to the specific implementation `XmlKeyManager` can you cast to the deletable interface `IDeletableKeyManager`.

Reply:
Thanks for the feedback. Changes done in latest commit.

Also started discussion https://github.com/dotnet/aspnetcore/discussions/60315 in ASP.NET Core repo.